src/features/imageUpload/imageUpload.js

```diff
@@ -0,0 +1,40 @@
+export default {
+
+	doSingleImageUpload: function(base64ImageData, progress) {
+
+		return new Promise(function(resolve, reject) {
+			var xmlHttpRequest = new XMLHttpRequest();
```

Comment:
dont we have axios or something?

Reply:
Switched to using fetch. Should work on mobile browsers.